### PR TITLE
SQL-1358: Download center release feed

### PR DESCRIPTION
**Pull request by nbagnard**

### CODE REVIEW

File: mongo-odbc-downloads_template.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+  "manual_link": "https://www.mongodb.com/docs/atlas/data-federation/query/query-with-sql/",
+  "release_notes_link": "https://www.mongodb.com/docs/atlas/data-federation/query/sql/release-notes/",
+  "versions": [
+    {
+      "version": "{RELEASE_VERSION}",
+      "platform": [
+        {
+          "name": "Windows",
+          "download_link": "https://translators-connectors-releases.s3.us-east-1.amazonaws.com/mongosql-odbc-driver/windows/{RELEASE_VERSION}/pkg/mongoodbc.msi"
```
Comment (pmeredit) [inline mongo-odbc-downloads_template.json:10]: Let's change "pkg" to "release"

**Review by pmeredit — APPROVED** [review]
LGTM! Just change "pkg" to "release" so that we can just be consistent across platforms. I believe I already updated that.

File: evergreen.yml [hunk 2]
```diff
@@ -1490,6 +1518,7 @@ tasks:
     commands:
       - func: "upload release"
       - func: "upload debug"
+      - func: "update download center feed"
```
Comment (bucaojit) [inline evergreen.yml:1521]: The `downloadcenter-update` task is being called in the `release` buildvariant, this one seems unnecessary.

**Review by bucaojit — APPROVED** [review]
Looks good.  Had a few comments which are similar to the ones in SQL-1357.

File: mongo-odbc-downloads_template.json [hunk 3]
```diff
@@ -0,0 +1,19 @@
+{
+  "manual_link": "https://www.mongodb.com/docs/atlas/data-federation/query/query-with-sql/",
+  "release_notes_link": "https://www.mongodb.com/docs/atlas/data-federation/query/sql/release-notes/",
```
[the remark this answers is not in the document]
Comment (nbagnard) [reply mongo-odbc-downloads_template.json:3]: Yes, the link changed sometimes in the last 2 weeks. I'll have to update it everywhere.

File: mongo-odbc-downloads_template.json (hunk 1 above)
Comment (nbagnard) [reply mongo-odbc-downloads_template.json:10]: @pmeredit 
It's not yet uploading to "release" from what I can see on master.
Do you know which PR is making that change?

**Review by bucaojit — APPROVED** [review]
LGTM

**Comment by nbagnard** [comment]
evergreen merge

**Pull request merged**